src/Type/Php/IntdivThrowTypeExtension.php

```diff
@@ -29,39 +28,19 @@ public function getThrowTypeFromFunctionCall(FunctionReflection $functionReflect
 			return $functionReflection->getThrowType();
 		}
 
-		$containsMin = false;
-		$valueType = $scope->getType($funcCall->getArgs()[0]->value);
-		foreach ($valueType->getConstantScalarTypes() as $constantScalarType) {
-			if ($constantScalarType->getValue() === PHP_INT_MIN) {
-				$containsMin = true;
-			}
-
-			$valueType = TypeCombinator::remove($valueType, $constantScalarType);
-		}
-
-		if (!$valueType instanceof NeverType) {
-			$containsMin = true;
-		}
+		$valueType = $scope->getType($funcCall->getArgs()[0]->value)->toInteger();
+		$containsMin = $valueType->acceptsWithReason(new ConstantIntegerType(PHP_INT_MIN), true);
```

Comment:
isSuperTypeOf() should be used instead of accepts()

Reply:
Updated.

Out of curiosity, what's the diff. Seems like both works.

Reply:
https://github.com/phpstan/phpstan/discussions/11460

Reply:
Thanks